.github/workflows/test.yml

```diff
@@ -18,78 +18,42 @@ jobs:
     runs-on: ubuntu-latest
 
     services:
-      mysql:
-        image: mysql:5.6
-        ports:
-          - 3306:3306
-        env:
-          MYSQL_ALLOW_EMPTY_PASSWORD: yes
-        options: --health-cmd="mysqladmin ping" --health-interval=10s --health-timeout=5s --health-retries=5
-        volumes:
-          - ${{ github.workspace }}/db/mysql:/var/lib/mysql
-
       mongo:
         image: mongo:4.0.6
         ports:
           - 27017:27017
         options: --health-cmd="mongo" --health-interval=10s --health-timeout=5s --health-retries=5
-        volumes:
-          - ${{ github.workspace }}/db/mongo:/data/db
 
     steps:
-      - name: verify mysql
-        run: mysql --host 127.0.0.1 --port 3306 -uroot -e "SHOW DATABASES"
-
       - uses: actions/checkout@v3
         with:
           path: client-repo
 
-      - name: 🐍 - Set up Python 3.6
-        uses: actions/setup-python@v2
+      - name: 🐍 - Set up Python 3.8
+        uses: actions/setup-python@v4
         with:
-          python-version: 3.6
-
-#      - uses: actions/cache@v3
-#        id: dbcache
-#        with:
-#          path: |
-#            ${{ github.workspace }}/db-cache
-#            !${{ github.workspace }}/db-cache/mongo/diagnostic.data
-#          key: ${{ hashFiles('client-repo/ci/*') }}
-
-#      - name: 💼 - Place cached fixtures
-#        if: steps.dbcache.outputs.cache-hit == 'true'
-#        run: |
-#          sudo cp -r ${{ github.workspace }}/db-cache ${{ github.workspace }}/db
+          python-version: 3.8
 
       - name: 🔧🔬 - Install EMG API
         run: |
           pip install --user --upgrade pip setuptools
           python -V
           pip -V
-          pip install "git+https://github.com/EBI-metagenomics/emgapi@develop#egg=emgcli"
+          pip install "git+https://github.com/EBI-metagenomics/emgapi@improve-local-dev#egg=emgcli"
```

Comment:
We need to changes this one after the merge (?)

Reply:
Yup. Just so that the integration tests can run properly on this branch. I had a trick before to use the same branch names between repos... but it didn't work with `feature/...` branch names. One day I might move the e2e tests to `mgnify-web` repo, and instead write more component tests on this repo. But that is a bigger task for another day.